src/tests/query.test.ts

```diff
@@ -3,42 +3,47 @@ import { getAPI } from './setupTests.js'
 describe('QueryTests', () => {
     const api = getAPI()
 
-    let query =
+    let genquery =
         "SELECT USER_NAME, USER_TYPE, USER_ZONE WHERE USER_TYPE != 'RODSGROUP'"
 
+    let specific_query =
+        "select token_id from R_TOKN_MAIN where token_name = 'rodsgroup'"
+    let specific_query_name = 'get_rodsgroup_token_id'
+
     beforeAll(async () => {
         await api.authenticate()
     })
 
     test('Execute genquery', async () => {
         const res = await api.queries.execute_genquery({
```

Comment:
Open an issue to rename `queries` to `query` _(to match the name of the endpoint)_.

Reply:
Done.